wallet/src/main/scala/org/bitcoins/wallet/models/UTXORecord.scala

```diff
@@ -0,0 +1,97 @@
+package org.bitcoins.wallet.models
+
+import org.bitcoins.core.currency.CurrencyUnit
+import org.bitcoins.core.hd.{
+  HDPath,
+  LegacyHDPath,
+  NestedSegWitHDPath,
+  SegWitHDPath
+}
+import org.bitcoins.core.protocol.script.{
+  ScriptPubKey,
+  ScriptWitness,
+  WitnessScriptPubKey
+}
+import org.bitcoins.core.protocol.transaction.{
+  TransactionOutPoint,
+  TransactionOutput
+}
+import org.bitcoins.core.wallet.utxo.TxoState
+import org.bitcoins.crypto.DoubleSha256DigestBE
+import org.bitcoins.db.DbRowAutoInc
+
+case class UTXORecord(
+    outpoint: TransactionOutPoint,
+    txid: DoubleSha256DigestBE, // TXID
+    state: TxoState, // state
+    scriptPubKeyId: Long, // output SPK
+    value: CurrencyUnit, // output value
+    path: HDPath,
+    redeemScript: Option[ScriptPubKey], // ReedemScript
+    scriptWitness: Option[ScriptWitness],
```

Comment:
This being here and on `AddressRecord` make me think that maybe the script witness and redeem script options belong in the SPK table instead of redundantly elsewhere